components/conversations/ThreadActivity.js

```diff
@@ -163,7 +163,10 @@ const ThreadActivity = ({ activity }) => {
         </Flex>
       )}
       {MESSAGES[activity.type] && (
-        <ActivityMessage color={activityColor}>{formatMessage(MESSAGES[activity.type])}</ActivityMessage>
+        <ActivityMessage color={activityColor}>
+          {formatMessage(MESSAGES[activity.type])}
+          {activity.data?.error?.message ? `: ${activity.data?.error?.message}` : ''}
```

Comment:
No need for it there

```suggestion
          {activity.data?.error?.message ? `: ${activity.data.error.message}` : ''}
```